scripts/labcontrol

```diff
@@ -15,6 +15,43 @@ import click
 from labcontrol.db.environment import patch_database
 
 
+import logging
+from os import environ
+
+
+debug_levels_list = {'DEBUG': logging.DEBUG,
+                     'INFO': logging.INFO,
+                     'WARNING': logging.WARNING,
+                     'ERROR': logging.ERROR,
+                     'CRITICAL': logging.CRITICAL}
+
+# the default logging level
+logging_level = logging.CRITICAL
+
+# the user can change the root logger's level using the LABCONTROL_DEBUG_LEVEL
+# environment variable. It must be set to a valid string in debug_levels_list,
+# or else an error will be raised. If this environment level is not set, then
+# logging level defaults to CRITICAL, the least verbose option.
+if 'LABCONTROL_DEBUG_LEVEL' in environ:
+    level = environ['LABCONTROL_DEBUG_LEVEL']
+    if level in debug_levels_list:
```

Comment:
How about the `try ... except` structure, which should be cheaper.

Reply:
This is more explicit, IMHO. Not sure if it would be cheaper than a conditional, either.

Reply:
I googled a bit and found diverse opinions. Here's the official explanation: [EAFP](https://docs.python.org/3/glossary.html#term-eafp). Totally trivial. Never mind:)

Reply:
I do like it better aesthetically...